src/foam/box/HTTPBox.js

```diff
@@ -157,11 +157,11 @@ try {
   java.io.OutputStreamWriter output = new java.io.OutputStreamWriter(conn.getOutputStream(),
                                                                      java.nio.charset.StandardCharsets.UTF_8);
 
-
-  output.write(new Outputter().stringify(message));
+  Outputter outputter = new Outputter();
```

Comment:
Would be better if Outputter had a constructor that took a Context then you could follow the structure of the original code.

Reply:
@kgrgreer I'll add it later. Can this be merged?